tests/js/app/testArticleHTMLRenderer.js

```diff
@@ -6,13 +6,33 @@ const SearchUtils = imports.search.utils;
 const SetMap = imports.app.setMap;
 
 describe('Article HTML Renderer', function () {
-    let wikihow_model, wikibooks_model;
+    let wikihow_model, wikibooks_model, wikipedia_model, wikisource_model;
     let renderer;
 
     beforeEach(function () {
         Utils.register_gresource();
 
         renderer = new ArticleHTMLRenderer.ArticleHTMLRenderer();
+        wikipedia_model = Eknc.ArticleObjectModel.new_from_props({
```

Comment:
CR pass on this commit too, but it doesn't apply without the other commits. I'd merge it if you ported it to use the old API